stock_move_value/models/stock_move.py

```diff
@@ -40,7 +40,7 @@ class StockMove(models.Model):
     def _compute_move_value(self):
         for move in self:
             # There can be multiple svls per move in case landed costs are entered
-            move.move_value = sum(move.stock_valuation_layer_ids.mapped("value"))
+            move.move_value = sum(move.sudo().stock_valuation_layer_ids.mapped("value"))
```

Comment:
Can we change `for move in self:` above to `for move in self.sudo():` instead?

Reply:
Please disregard this. It's not quite right to add `sudo()` for the entire `self`.